src/api/auth/auth.post.test.ts

```diff
@@ -35,6 +36,23 @@ describe('User Authentication', () => {
     expect(user.birthDate).to.equal(dbUser.birthDate.toISOString());
   });
 
+  it('should have extended token duration', async () => {
+    const response = await axios.post('http://localhost:8080/auth', {
+      email: USER_TO_CREATE.email,
+      password: USER_TO_CREATE.password,
+      rememberMe: true,
+    });
+
+    expect(response.status).to.equal(200);
+    expect(response.data).to.have.property('token');
+
+    const decodedToken = jwt.verify(response.data.token, process.env.JWT_SECRET);
+    assert(typeof decodedToken !== 'string');
+
+    const currentTime = Math.floor(Date.now() / 1000);
+    expect(decodedToken.exp).to.be.equal(currentTime + 60 * 60 * 24 * 7);
```

Comment:
Inclusive vc poderia exportar aquelas constantes e reutilizá-las aqui, porque assim se vc quiser mexer no tempo de expiração do token, você não precisa atualizar o teste.

**OBS:** tem coisas que nós **queremos** propositalmente forçar que seja mexido no teste após a mudança no código, por boa prática. Mas valores constantes como esse eu acho que não faz muito sentido, pois são escolhas de negócio e não alteram nenhum comportamento, apenas o valor em si.